testing/file_test/testdata/lsp_keywords.carbon

```diff
@@ -16,39 +16,53 @@
 multi
 line
 ]]
+[[@LSP-CALL:bar:content]]
+[[@LSP-CALL:baz:
+multi
+line]]
 [[@LSP-REPLY:7]]
 [[@LSP-REPLY:8:bar]]
 [[@LSP-NOTIFY:exit]]
 
 // --- AUTOUPDATE-SPLIT
 
 // CHECK:STDERR: --- STDIN:
-// CHECK:STDERR: Content-Length: 48
+// CHECK:STDERR: Content-Length: 37
 // CHECK:STDERR:
-// CHECK:STDERR: {"jsonrpc": "2.0", "id": "1", "method": "foo"}
+// CHECK:STDERR: {"jsonrpc": "2.0", "method": "foo"}
 // CHECK:STDERR:
-// CHECK:STDERR: Content-Length: 48
+// CHECK:STDERR: Content-Length: 37
 // CHECK:STDERR:
-// CHECK:STDERR: {"jsonrpc": "2.0", "id": "2", "method": "foo"}
+// CHECK:STDERR: {"jsonrpc": "2.0", "method": "foo"}
 // CHECK:STDERR:
-// CHECK:STDERR: Content-Length: 57
+// CHECK:STDERR: Content-Length: 46
 // CHECK:STDERR:
-// CHECK:STDERR: {"jsonrpc": "2.0", "id": "3", "method": "bar", content}
+// CHECK:STDERR: {"jsonrpc": "2.0", "method": "bar", content}
 // CHECK:STDERR:
-// CHECK:STDERR: Content-Length: 61
+// CHECK:STDERR: Content-Length: 50
 // CHECK:STDERR:
-// CHECK:STDERR: {"jsonrpc": "2.0", "id": "4", "method": "baz",
+// CHECK:STDERR: {"jsonrpc": "2.0", "method": "baz",
 // CHECK:STDERR: multi
 // CHECK:STDERR: line
 // CHECK:STDERR: }
 // CHECK:STDERR:
+// CHECK:STDERR: Content-Length: 67
+// CHECK:STDERR:
+// CHECK:STDERR: {"jsonrpc": "2.0", "method": "bar", "id": 1, "params": {content}}
+// CHECK:STDERR:
+// CHECK:STDERR: Content-Length: 71
+// CHECK:STDERR:
+// CHECK:STDERR: {"jsonrpc": "2.0", "method": "baz", "id": 2, "params": {
+// CHECK:STDERR: multi
+// CHECK:STDERR: line}}
+// CHECK:STDERR:
 // CHECK:STDERR: Content-Length: 31
 // CHECK:STDERR:
 // CHECK:STDERR: {"jsonrpc": "2.0", "id": "7"}
 // CHECK:STDERR:
-// CHECK:STDERR: Content-Length: 36
+// CHECK:STDERR: Content-Length: 48
 // CHECK:STDERR:
-// CHECK:STDERR: {"jsonrpc": "2.0", "id": "8", bar}
+// CHECK:STDERR: {"jsonrpc": "2.0", "id": "8", "result": {bar}}
```

Comment:
LSP-REPLY still puts quotes (`"`) around the id value, though LSP-CALL does not.

Reply:
That's because `LSP-CALL` is generating the value and guarantees it's an int, whereas `LSP-REPLY` has it as user input. I'll add a comment explaining this.